lstchain/data/data_selection_cuts.json

```diff
@@ -0,0 +1,15 @@
+{
+  "general":{
+    "gh_score": 0.8,
+    "source_fov_offset": 2.83,
+    "intensity": 200,
+    "r": 1,
+    "wl": 0.1,
+    "leakage2_intensity": 0.2,
+    "leakage_intensity_width_2": 0.2,
+    "alpha": 8
+    },
+  "mc":{
+    "theta_cut": 0.14
```

Comment:
I would highly recommend to also use units in the config file. This could be achieved like this:

```
"theta_cut": {"value": 0.14, "unit": "deg"}
```

And then reading with:

```
theta_cut = u.Quantity(**config["theta_cut"])
```

Reply:
Ok thanks! I will make the change.

Reply:
Using units in this config file, would not let me use filter_events as it is. Would you then recommend to make this config file like the lstchain_standard_config to suit filter_events? As only the angular_offset parameters need units, so those cuts can be just adjusted in the main script itself.